src/app.js

```diff
@@ -38,8 +37,8 @@ export default function (app) {
     }
 
     if (obj = plugin.hooks) {
-      Object.keys(obj).forEach(function (key) {
-        hooks[key].push(obj[key])
+      Object.keys(obj).map(function (key) {
```

Comment:
Agrees with @leeoniya sentiment.. there are more important factors to consider choosing `map` over `forEach` and vice versa besides the saving of 4 bytes